src/main/java/crawlercommons/utils/Strings.java

```diff
@@ -0,0 +1,28 @@
+package crawlercommons.utils;
+
+/**
+ * Util functions for manipulating strings.
+ */
+public class Strings {
+
+    public static boolean isBlank(final String cs) {
+        if (cs == null || cs.isEmpty()) {
+            return true;
+        }
+        for (int i = 0; i < cs.length(); i++) {
+            if (Character.isWhitespace(cs.charAt(i)) == false) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public static boolean isNotBlank(final String cs) {
```

Comment:
!sure this one is needed
it is clear enough to use !isBlank(...) in the code, no ?

Reply:
I don't have strong options here, so removed it.